Significantly offload scoring sections of the `Game mode` articles (#6607)
ppy/osu-wiki: wiki/Game_mode/osu!/en.md

## Scoring

[The osu! score](/wiki/Gameplay/Score/osu!) is a weighted sum of all hit objects' values. It depends on the following:

- [Judgement](/wiki/Gameplay/Judgement/osu!) determines an object's base scoring value (300, 100, 50, or 0 in case of a miss). For [hit circles](/wiki/Hit_object/Hit_circle), well-timed key presses are valued more, both in terms of score and accuracy. [Sliders](/wiki/Hit_object/Slider) and [spinners](/wiki/Hit_object/Spinner) don't have hit windows, but will break active combo, possibly with a miss, if not cleared properly. Objects with higher judgement also provide higher [health](/wiki/Beatmapping/Health) boost.
- [Accuracy](/wiki/Gameplay/Accuracy#osu!) depends on judgement and shows how precise hits are. Late or early key presses, as well as misses, decrease overall accuracy.

> - [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: the higher it is, the more the object contributes to the total score. Combo may be [broken](/wiki/Glossary/Combobreak) by a miss or a [slider break](/wiki/Gameplay/Slider_break).

Reviewer: ```suggestion
- [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: hitting a hit object contributes more to the total score when combo is high and vice versa. Combo may be [broken](/wiki/Glossary/Combobreak) by a miss or a [slider break](/wiki/Gameplay/Slider_break).
```

> - [Combo](/wiki/Gameplay/Combo_(score_multiplier)) is a score multiplier: clearing a hit object contributes more to the total score when combo is high and vice versa. Combo may be [broken](/wiki/Glossary/Combobreak) by a miss or a [slider break](/wiki/Gameplay/Slider_break).

Reviewer: use this wording on the other modes?

Author: right. should be done now

> When combo is maintained, the total score grows exponentially. Objects closer to the end of the map are worth orders of magnitude more points than the ones in the beginning, which means that a player will lose way more score on them in case of mistimed hits. As a result, it's possible and very common for a score with lower accuracy to have higher amount of points and beat the score with higher accuracy.

Reviewer: because score doesn't actually decrease

```suggestion
When combo is maintained, the total score grows exponentially. Objects closer to the end of the map are worth orders of magnitude more points than the ones in the beginning, which means that a player will lose way more potential score on them in case of mistimed hits. As a result, it's possible and very common for a score with lower accuracy to have a higher amount of points and beat a score with higher accuracy.
```

> After the beatmap is over, the score is assigned a [grade](/wiki/Gameplay/Grade#osu!), a short accuracy assessment in form of a single letter. A golden or silver SS denotes 100% accuracy, and everything else, from S to D, depends on amount of 300s, 50s, and misses.

Reviewer: ```suggestion
After completing a beatmap, the score is assigned a [grade](/wiki/Gameplay/Grade#osu!), a short accuracy assessment in the form of a single letter. A golden or silver SS denotes 100% accuracy, and everything else, from S to D, depends on amount of 300s, 50s, and misses.
```

Reviewer: wuh, no?

> After completing a beatmap, the score is assigned a [grade](/wiki/Gameplay/Grade#osu!), a short accuracy assessment in the form of a single letter. A golden or silver SS denotes 100% accuracy, and everything else, from S to D, depends on amount of 300s, 50s, and misses.

Reviewer: ```suggestion
After completing a beatmap, the score is assigned a [grade](/wiki/Gameplay/Grade#osu!), a short accuracy assessment in the form of a single letter. A golden or silver SS denotes 100% accuracy, and everything else, from S to D, depends on the amount of 300s, 50s, and misses.
```
